Automattic/vip-go-mu-plugins
# Remove test from the Site Health tool page

## Description

The Site Health tool page is showing some critical/failed issues for tests that aren't relevant to VIP:
<img width="743" alt="site health" src="https://user-images.githubusercontent.com/29221840/77514043-4e0bb900-6e33-11ea-951d-54f30ea90ba7.png">

This PR removes the unnecessary `background_updates` test.

Here's a list of all tests run for the Site Health tool in WP core:
https://core.trac.wordpress.org/browser/tags/5.3/src/wp-admin/includes/class-wp-site-health.php#L1842

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Go to `wp-admin` > `Tools` > `Site Health`
1. Verify `Background updates` error/issue isn't present:
<img width="1519" alt="site health after" src="https://user-images.githubusercontent.com/29221840/77589858-b39a8c80-6ea9-11ea-87d1-60a8c62b1d98.png">
